scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#to be executed from the root project directory
+git rev-parse --short HEAD > version.txt
```

Comment:
> The version that lands in the version.txt file is connected more with the version of the UI and only the UI uses it so maybe it should land in the shiny directory? A side effect of such an approach would be that when deploying the shiny application it would be sufficient to copy the contents of the shiny directory without worrying about this additional special file.
> 
> In fact git version is related to both UI and R package since they are both in the same repository. But it seems to be a good idea and simplifies deployment process. Let's do it.

Ok.